src/tests/receiving/validationsOnDeliverOnDate.test.ts

```diff
@@ -0,0 +1,102 @@
+import { ShipmentType } from '@/constants/ShipmentType';
+import { expect, test } from '@/fixtures/fixtures';
+import { StockMovementResponse } from '@/types';
+import { getDateByOffset } from '@/utils/DateUtils';
+
+test.describe('Validations on edit Deliver On Date when receiving shipment', () => {
+  let STOCK_MOVEMENT: StockMovementResponse;
+
+  test.beforeEach(
+    async ({
+      supplierLocationService,
+      stockMovementService,
+      mainProductService,
+    }) => {
+      const supplierLocation = await supplierLocationService.getLocation();
+      STOCK_MOVEMENT = await stockMovementService.createInbound({
+        originId: supplierLocation.id,
+      });
+
+      const product = await mainProductService.getProduct();
+
+      await stockMovementService.addItemsToInboundStockMovement(
+        STOCK_MOVEMENT.id,
+        [{ productId: product.id, quantity: 50 }]
+      );
+
+      await stockMovementService.sendInboundStockMovement(STOCK_MOVEMENT.id, {
+        shipmentType: ShipmentType.AIR,
+      });
+    }
+  );
+
+  test.afterEach(async ({ stockMovementShowPage, stockMovementService }) => {
+    await stockMovementShowPage.goToPage(STOCK_MOVEMENT.id);
```

Comment:
don't we need a null safe here, in case the stock movement has not been created successfully in the `beforeEach`?
I assume the test would fail, but it would not be the test that failed, but the error saying "cannot read property id of undefined"